src/js/jsx/shared/Datalist.jsx

```diff
@@ -41,7 +43,8 @@ define(function (require, exports, module) {
             options: React.PropTypes.instanceOf(Immutable.Iterable),
             live: React.PropTypes.bool,
             startFocused: React.PropTypes.bool,
-            placeholderText: React.PropTypes.string
+            placeholderText: React.PropTypes.string,
+            useAutofill: React.PropTypes.bool
```

Comment:
Can you document somewhere what this new property does? We should be better generally about document state and props in React components...